src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImpl.java

```diff
@@ -270,21 +270,72 @@ private CollectionExercise newCollectionExerciseFromDTO(CollectionExerciseDTO co
         collectionExercise.setState(CollectionExerciseDTO.CollectionExerciseState.CREATED);
         collectionExercise.setCreated(new Timestamp(new Date().getTime()));
         collectionExercise.setId(UUID.randomUUID());
-
-        log.debug("Successfully created collection exercise from DTO");
+        log.debug("Successfully created collection exercise from DTO, CollectionExerciseId: {}",
+                collectionExercise.getId());
         return collectionExercise;
     }
 
     /**
-     * Set up data for creation of action plans
+     * Create required action plans
      * @param collectionExercise Collection Exercise
      * @param survey SurveyDTO representing survey of collection exercise
      */
     private void createActionPlans(CollectionExercise collectionExercise, SurveyDTO survey) {
-        log.debug("Creating action plans, CollectionExerciseId: {}", collectionExercise.getId());
-        createOverrideActionPlan(collectionExercise, survey,  "B");
-        createOverrideActionPlan(collectionExercise, survey,  "BI");
-        log.debug("Successfully created action plans, CollectionExerciseId: {}", collectionExercise.getId());
+        log.debug("Creating action plans for exercise, CollectionExerciseId: {}, SurveyId: {}",
+                collectionExercise.getId(), survey.getId());
+        createDefaultActionPlan(survey, "B");
+        createDefaultActionPlan(survey, "BI");
+        createOverrideActionPlan(collectionExercise, survey, "B");
+        createOverrideActionPlan(collectionExercise, survey, "BI");
+        log.debug("Successfully created action plans for exercise, CollectionExerciseId: {}, SurveyID: {}",
+                collectionExercise.getId(), survey.getId());
+    }
+
+    /**
+     * Create default action plan for collection exercise and case type if not already exists
+     * @param survey DTO Representation of survey
+     * @param sampleUnitType Sample Unit Type i.e. (B, H, HI)
+     */
+    private void createDefaultActionPlan(SurveyDTO survey, String sampleUnitType) {
+        log.debug("Creating default action plan, SurveyId: {} , SampleUnitType: {}",
+                survey.getId(), sampleUnitType);
+
+        // If a casetypedefault already exists for this survey/sampleUnitType do nothing
+        CaseTypeDefault existingCaseTypeDefault = caseTypeDefaultRepo.findTopBySurveyIdAndSampleUnitTypeFK(
+                UUID.fromString(survey.getId()), sampleUnitType);
+        if (existingCaseTypeDefault != null) {
+            log.debug("Default action plan already exists, SurveyId: {} SampleUnitType: {}",
+                    survey.getId(), sampleUnitType);
+            return;
+        }
+
+        // Create new action plan and associated casetypedefault
+        String shortName = survey.getShortName();
+        String name = String.format("%s %s", shortName, sampleUnitType);
+        String description = String.format("%s %s Case", shortName, sampleUnitType);
```

Comment:
I think the reason for doing this is because we have to pass the `name` and `description` into the `createActionPlan` function and it could be a bit messy to do that inline when calling the function

Reply:
I disagree as it add description, however, this is completely stylistic choice.